### PR TITLE
GH: update auto sync to use aormsby/Fork-Sync-With-Upstream-action

### PR DESCRIPTION
**Pull request by zdtsw**
To auto sync branches from upstream to downstream without creating new token

### CODE REVIEW

File: .github/workflows/sync-release-branch-2023a.yml [hunk 1]
```diff
@@ -7,27 +7,31 @@ on:
    - cron: "0 13 * * 6"
```
Comment (atheo89) [inline .github/workflows/sync-release-branch-2023a.yml:7]: May I ask you to change this cron to `0 8 * * 2` ? (Since we have this PR open :) )

Comment (zdtsw) [reply .github/workflows/sync-release-branch-2023a.yml:7]: sure, if `0 8 * * 2` works better than old cronjob

**Pull request merged**